social_panel/lib/twitter_client.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'faraday'
+require 'JSON'
+require 'pry'
+
+class TwitterClient
+  URL = 'https://api.twitter.com/1.1'
+  ACCESS_TOKEN = "Bearer #{ENV['SOCIAL_PANEL_TWITTER_ACCESS_TOKEN']}"
```

Comment:
Naming:

I'd say `ENV['SOCIAL_PANEL_TWITTER_ACCESS_TOKEN']` is the access token and `"Bearer #{ENV['SOCIAL_PANEL_TWITTER_ACCESS_TOKEN']}"` is the `AUTHORIZATION_HEADER`.